### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by dbnicholson**
I started linting a few files and then I figured I should just go all the way.

### CODE REVIEW

**Review by wjt — APPROVED** [review]
LGTM, I always like more linters. My only concern is one of line length.

File: helpers/assemble-manifest [hunk 1]
```diff
@@ -7,10 +7,12 @@ import json
 import os
 import sys
 
+
 def dict_merge(target, source):
     """Deep merge of dictionaries
 
-    See http://blog.impressiver.com/post/31434674390/deep-merge-multiple-python-dicts
```
[the remark this answers is not in the document]
Comment (dbnicholson) [reply helpers/assemble-manifest]: I changed that one line but didn't go back to find any others that were originally between 79 and 88.

**Comment by dbnicholson** [comment]
Care to look at the `.editorconfig` file I added? I think it's right, but an extra eye would be nice. I've never actually used it before, but it seems a whole lot better than adding an emacs config that only I'll use or a vim config that I'll never use.

**Comment by wjt** [comment]
LGTM!

**Pull request merged**